app/entry.js

```diff
@@ -1,6 +1,8 @@
-const path = require('path')
-require('./lib/report/node.js')
+import { join as pathJoin } from 'path'
+import './lib/report/node.js'
 
+import { checkForAddRequests } from './setup-add-requests'
+checkForAddRequests()
```

Comment:
add a small comment also here so in case we get over this code we will understand what this line is doing